configuration/backend_configuration/locationtags/locationtags.csv

```diff
@@ -5,3 +5,4 @@ a2327745-2970-4752-ac8a-dd0ba131f40e,,Facility Location,
 f5b9737b-14d5-402b-8475-dd558808e172,,Admission Location,Patients may only be admitted to inpatient care in a location with this tag
 9783aba6-df7b-4969-be6e-1e03e7a08965,,Transfer Location,Patients may only be transfer to inpatient care in a location with this tag
 e2a1b3c4-d5e6-7890-1abc-def234567890,,Visit Location,A location where a patient may have a visit
+b5d01d4f-e188-4c6e-a220-c2263b849ad2,,Appointment Location,"When a user user creates a appointment service and chooses a location, they may only choose one with this tag"
```

Comment:
nit: Maybe one of the following tag definitions is a touch clearer?

- Appointments may only be created and scheduled at locations with this tag
- A location where appointment services can be created and scheduled

Reply:
Done